test/integration/targets/docker_swarm_info/tasks/test_swarm_info.yml

```diff
@@ -125,6 +125,39 @@
          - 'output.docker_swarm_active == true'
          - 'output.docker_swarm_manager == true'
 
+  - name: Try to get docker_swarm_info and swarm_unlock_key on non a unlocked swarm
+    docker_swarm_info:
+      unlock_key: yes
+    register: output
+
+  - name: assert reading swarm facts and non existing swarm unlock key
+    assert:
+      that:
+        - 'not output.swarm_unlock_key'
```

Comment:
```suggestion
        - 'output.swarm_unlock_key is none'
```

Reply:
And maybe in some of the tests above: `output.swarm_unlock_key is not defined`

Reply:
The returned value from the Docker API actually is an empty string, hence the `not` assertion. But maybe it makes a cleaner API to set `swarm_unlock_key` to `None` if it's an empty string.

Reply:
Something like:

```python
    def get_docker_swarm_unlock_key(self):
        unlock_key = self.client.get_unlock_key()
        return unlock_key.get('UnlockKey') or None
```